sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -675,7 +675,13 @@ def collect(
         Timestream
             Timestream containing the collected list at each point.
         """
-        return _aggregation("collect", self, window, max, min)
+        if pa.types.is_list(self.data_type):
+            print("collecting list")
+            input = record({"value": self})
```

Comment:
Could also write as:

```
return record({"value": self}).collect(window=window, max=max, min=min).col("value")
```

(Eg., using collect rather than `_aggregation`). Not sure which is better.